plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -1539,6 +1539,17 @@ public void testTableWithTrailingSlashLocation(boolean partitioned)
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testAnonymousRowTypeFailure()
+    {
+        String tableName = "test_table_anonymous_row_type_" + randomNameSuffix();
+        assertQueryFails("CREATE TABLE " + tableName + " AS SELECT ROW(123) AS c1", "Anonymous row type is not supported in Delta Lake connector");
```

Comment:
Could you try including the issue column name in the error message? The name is helpful in case of a wide column table.

Reply:
Added issue column name to the error message.
now the message looks like:
"Anonymous row type is not supported in Delta Lake connector for column: c1"